core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data.service.spec.ts

```diff
@@ -21,8 +21,6 @@
 // thread-data.service.ts is upgraded to Angular 8.
 import { UpgradedServices } from 'services/UpgradedServices';
 
-import { TranslatorProviderForTests } from 'tests/test.extras';
-
 require('domain/feedback_thread/FeedbackThreadObjectFactory.ts');
```

Comment:
This check is not checking if we have unwanted requires in my opinion. I don't think we can have such check right now since there are require statements added in many files due to webpack compilation. After the angular migration is done we would not have requires anywhere but just in the e2e tests files. Then we will have a check to ensure no unused require is added.

However, I also think the unwanted requires might be removed when we add `unused-var` plugin because in the e2e tests we have variables like var waitFor = require('./waitFor'). So, if waitFor is not used we can remove that require.

@Hudda could you please add this check in the issue itself `remove unnecessary require from the codebase` and link this comment there?

Reply:
Done.

Reply:
:D